core/templates/dev/head/pages/exploration-editor-page/improvements-tab/services/improvement-modal.service.ts

```diff
@@ -248,6 +248,12 @@ angular.module('oppia').factory('ImprovementModalService', [
                     $scope.messageSendingInProgress = false;
                   }).then($uibModalInstance.close);
               };
+
+              $scope.onClickReviewSuggestion = function() {
+                $scope.messageSendingInProgress = true;
+                openSuggestionReviewer($scope.activeThread);
```

Comment:
Is it still necessary to define $scope.openSuggestionReviewer above?

Also, I actually have no idea what this function does, based on its name. Is it possible to improve the name of the function? (I don't know what it means to "open" the reviewer of a suggestion.)

Reply:
It opens the view to review the suggestion (I think that's what Brian meant by suggestionReviewer). I can remove the binding to the scope. Not sure of what to rename it to!

Reply:
openReviewerSuggestionModal / openReviewerSuggestionPage perhaps -- something like that? (Not sure what you mean by "view".)

Reply:
done. I meant the modal itself. I have renamed it.